_data/sites.json

```
@@ -4022,6 +4022,20 @@
         ]
     },
 
+    {
+        "name": "ExpressVPN",
+        "url": "https://www.expressvpn.com/support/",
+        "difficulty": "hard",
+        "notes": "You must either go through live support on the support page and specifically request the deletion of your account or write an Email from the registered Address. Either way you will probably be asked to confirm again by mail.",
+        "notes_de": "Auf Englisch kannst du entweder den Support Chat explizit um die Löschung deines Accounts bitten, oder eine Email von der zum Account gehörenden Adresse schicken. Wie du auch vorgehst wirst du vermutlich aufgefordert werden die Löschung per Mail zu bestätigen.",
+        "domains": [
+            "expressvpn.com"
+        ],
+        "email": "support@expressvpn.zendesk.com",
+        "email_subject": "Request to Delete my Account",
+        "email_body": "To whom it may concern,%0D%0A%0D%0AI am reaching out to you to request the cancellation and deletion of the ExpressVPN account attached to this Email address and its data as I am no longer using the service.%0D%0A%0D%0ABest Regards"
```

```suggestion
        "email": "support@expressvpn.zendesk.com",
        "email_subject": "Request to Delete my Account",
        "email_body": "To whom it may concern,%0D%0A%0D%0AI am reaching out to you to request the cancellation and deletion of the ExpressVPN account attached to this Email address and its data as I am no longer using the service.%0D%0A%0D%0ABest Regards",
        "domains": [
            "expressvpn.com"
        ]
```
Just a minor thing, I like to leave domains last.